fhir-server/liberty-config/config/default/fhir-server-config.json

```
@@ -56,6 +56,18 @@
                     "max.block.ms": "60000",
                     "max.in.flight.requests.per.connection": "5"
                 }
+            },
+            "nats": {
+                "enabled": false,
+                "cluster": "nats-streaming",
+                "channel": "fhirNotifications",
+                "clientId": "fhir-server",
+                "servers": "nats://nats-node1:4222,nats://nats-node2:4222,nats://nats-node3:4222",
```

docker-compose question:  if liberty can reach all the nodes on 4222, do we even need to map those in the default docker-compose?

for example, currently it has:
* 4222:4222
* 4223:4222
* 4224:4222

---

ok, after testing it, I'm think we should use a default config that matches the test:
```suggestion
                "servers": "nats://nats-node1:4222,nats://nats-node2:4223,nats://nats-node3:4224",
```

---

The idea behind using 4222 for all NATS servers is that, because the FHIR server is on the same docker network as NATS, we can use the internal port number for each NATS service, which is 4222 in each container.  I'm a bit surprised that the combination of service name + external port number (4222,4223,4224) works.

---

totally agree it should use the internal port (and not the external one).  but it seems to me like nats-node2 and nats-node3 are actually listening on internal ports 4223 and 4224 (and not 4222)